source/Engine.cpp

```diff
@@ -1032,7 +1032,7 @@ void Engine::Step(bool isActive)
 void Engine::Go()
 {
 	++step;
-	currentCalcBuffer = currentCalcBuffer ? 0 : 1;
+	currentCalcBuffer = (!currentCalcBuffer) & 1;
```

Comment:
this is just less readable
```suggestion
	currentCalcBuffer = !currentCalcBuffer;
```
should be the same?

Reply:
Probably. I don't know why I didn't try that.

Reply:
Well I do, It's because an earlier version of this line I had was `++currentCalBuffer & 1` so we increment then only take the last bit.